relayer/relays/beacon/config/config.go

```diff
@@ -35,8 +34,14 @@ type BeaconConfig struct {
 }
 
 type SinkConfig struct {
-	Parachain          config.ParachainConfig `mapstructure:"parachain"`
-	UpdateSlotInterval uint64                 `mapstructure:"updateSlotInterval"`
+	Parachain          ParachainConfig `mapstructure:"parachain"`
+	UpdateSlotInterval uint64          `mapstructure:"updateSlotInterval"`
+}
+
+type ParachainConfig struct {
+	Endpoint             string `mapstructure:"endpoint"`
+	MaxWatchedExtrinsics int64  `mapstructure:"maxWatchedExtrinsics"`
+	HeaderRedundancy     uint64 `mapstructure:"headerRedundancy"`
```

Comment:
Setting to capture the max headers in the ringbuffer: // https://github.com/paritytech/polkadot-sdk/blob/master/bridges/snowbridge/pallets/ethereum-client/src/lib.rs#L75

Reply:
Can we add a comment in the code describing the config item?

Reply:
Sure, https://github.com/Snowfork/snowbridge/pull/1294/commits/1956d878b677cd2a3b74b1d7313c27b3c15e6e87.